tools/nmr_bucketing/README.rst

```diff
@@ -0,0 +1,32 @@
+
+Changelog/News
+--------------
+
+**Version 2.0.2 - 21/04/2017**
+
+- IMPROVEMENT: Add “zoomed” representations of bucketed spectra, depending on exclusion zone borders
+
+**Version 1.0.3 - 24/10/2016**
+
+- ENHANCEMENT: add possibility of bucketing processed files (upstream tools)
+
+**Version 1.0.2 - 12/08/2016**
+
+- ENHANCEMENT: x-axis customization: add chemical shift labels 
+
+**Version 1.0.1 - 04/04/2016**
+
+- TEST: refactoring to pass planemo test using conda dependencies
+
+
+**Version 2015-01-08 - 08/01/2015**
+
+
+
+Test Status
+-----------
+
+Planemo test using conda: passed
+
+Planemo shed_test: passed
+
```

Comment:
I think this is not required as it doesn't really tell the user or admin anything.